platform/ios/jazzy.yml

```diff
@@ -15,7 +15,7 @@ umbrella_header: src/Mapbox.h
 framework_root: ../darwin/src
 
 custom_categories:
-  - name: Guides
+  - name: Appendices
```

Comment:
The macOS table of contents can also call them appendices. I can take care of that at release time if you don’t get to it first.

Reply:
I opened a new PR: https://github.com/mapbox/mapbox-gl-native/pull/12634